File: put-on-prod.rb
require "fileutils"
system("mvn clean install")
FileUtils.cp_r("dist/.", "#{ENV["QIN_ROOT"]}/Prod")
FileUtils.cp("start.bat", "#{ENV["QIN_ROOT"]}/Prod")
FileUtils.cp("start.sh", "#{ENV["QIN_ROOT"]}/Prod")

File: put-on-prod.py
#!/usr/bin/env python3

import os
import shutil

qin_root = os.environ['QIN_ROOT']
os.system("mvn clean install")
shutil.copytree("./dist", f"{qin_root}/Prod", dirs_exist_ok=True)
shutil.copy("./start.bat", f"{qin_root}/Prod/start.bat")
shutil.copy("./start.sh", f"{qin_root}/Prod/start.sh")
